pkg/api/controller.go

```diff
@@ -2937,9 +2937,14 @@ func (c *Controller) handleAPIErrorCallback(ctx context.Context, w http.Response
 	// Handle Hook Errors
 	var hookAbortErr *graveler.HookAbortError
 	if errors.As(err, &hookAbortErr) {
-		log.WithField("run_id", hookAbortErr.RunID).Warn("aborted by hooks")
-		cb(w, r, http.StatusPreconditionFailed, hookAbortErr.Unwrap())
-		return true
+		var clientErr *actions.HookClientError
```

Comment:
- if we use fmt to wrap the errors from the execution, we don't need this code.
also we like to report the full error not just the underlying one as we miss information about the action itself.